pkgs/by-name/py/python-ciel/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+}:
+
+let
+  buildPythonApplication = python3Packages.buildPythonApplication;
+in
+
```

Comment:
This is not needed once package is called from `pkgs/top-level/python-packages.nix``

Reply:
No, see comment from @eljamm.